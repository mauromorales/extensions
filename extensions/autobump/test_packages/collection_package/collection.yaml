packages:
  - name: "golang1"
    category: "build"
    version: 1.16.6+7
    base_url: https://golang.org/dl
    hidden: true # No need to make it installable for now
    labels:
      autobump.revdeps: "false"
      autobump.string_replace: '{ "prefix": "" }'
      autobump.strategy: "custom"
      autobump.prefix: "prefix"
      autobump.hook: |
        echo "1.21.3"
      autobump.version_hook: |
        echo "1.21.3"
      package.version: "1.16.6"
      autobump.checksum_hook: "curl -q -L https://storage.googleapis.com/golang/go{{.Values.labels.package.version}}.linux-amd64.tar.gz.sha256"
      package.checksum: "be333ef18b3016e9d7cb7b1ff1fdb0cac800ca0be4cf2290fe613b3d069dfe0d"
  - name: "golang2"
    category: "build"
    version: 1.16.6+7
    base_url: https://golang.org/dl
    hidden: true # No need to make it installable for now
    labels:
      autobump.revdeps: "false"
      autobump.string_replace: '{ "prefix": "" }'
      autobump.strategy: "custom"
      autobump.prefix: "prefix"
      autobump.hook: |
        echo "1.21.3"
      autobump.version_hook: |
        echo "1.21.3"
      package.version: "1.16.6"
      autobump.checksum_hook: "curl -q -L https://storage.googleapis.com/golang/go{{.Values.labels.package.version}}.linux-amd64.tar.gz.sha256"
      package.checksum: "be333ef18b3016e9d7cb7b1ff1fdb0cac800ca0be4cf2290fe613b3d069dfe0d"