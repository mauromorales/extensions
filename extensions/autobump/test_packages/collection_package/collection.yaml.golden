packages:
  - name: "golang1"
    category: "build"
    version: "1.21.3"
    base_url: https://golang.org/dl
    hidden: true # No need to make it installable for now
    labels:
      autobump.revdeps: "false"
      autobump.string_replace: '{ "prefix": "" }'
      autobump.strategy: "custom"
      autobump.prefix: "prefix"
      autobump.hook: |
        echo "1.21.3"
      autobump.version_hook: |
        echo "1.21.3"
      package.version: "1.21.3"
      autobump.checksum_hook: "curl -q -L https://storage.googleapis.com/golang/go{{.Values.labels.package.version}}.linux-amd64.tar.gz.sha256"
      package.checksum: "1241381b2843fae5a9707eec1f8fb2ef94d827990582c7c7c32f5bdfbfd420c8"
  - name: "golang2"
    category: "build"
    version: "1.21.3"
    base_url: https://golang.org/dl
    hidden: true # No need to make it installable for now
    labels:
      autobump.revdeps: "false"
      autobump.string_replace: '{ "prefix": "" }'
      autobump.strategy: "custom"
      autobump.prefix: "prefix"
      autobump.hook: |
        echo "1.21.3"
      autobump.version_hook: |
        echo "1.21.3"
      package.version: "1.21.3"
      autobump.checksum_hook: "curl -q -L https://storage.googleapis.com/golang/go{{.Values.labels.package.version}}.linux-amd64.tar.gz.sha256"
      package.checksum: "1241381b2843fae5a9707eec1f8fb2ef94d827990582c7c7c32f5bdfbfd420c8"
